### DISCUSSION TITLE
element type check for long list

### DISCUSSION THREAD

**Discussion by pinterior** [microsoft/pyright#8314] [Q&A] [answered]
currently, code like below does not produce any errors.

https://pyright-play.net/?locale=en&code=IYLgBANglgzgLgbSgOzgXTAXjAgUGAsABgBowBGMgJjIGYyAWMgVjIDYyB2MgDjIE4S%2BQqQrU6jFuy68BQwsTKUwNMPTBMwrMBzDcwfMIOEFRy1es3bd%2Bw8YVnxayVul7ZR%2BSKVPLUnTIGciaKYioSYABEkWi4QA

it seems type check is not performed to 64th or later elements.
is there any way (command line or configuration option) to increase the limit?
or how about emitting warning like "some of elements are not checked"?

**Comment by erictraut** [comment, marked as answer]
Yes, this is a bug. It will be addressed in the next release.
